webapp/sources/rudder/rudder-web/src/main/webapp/secure/nodeManager/searchNodes.html

```diff
@@ -17,6 +17,8 @@
   <script type="text/javascript" data-lift="with-cached-resource" src="/javascript/rudder/angular/auditMode.js"></script>
   <script type="text/javascript" data-lift="with-cached-resource" src="/javascript/rudder/angular/nodeProperties.js"></script>
   </head>
+
+  <div data-lift="node.SearchNodes.head"></div>
```

Comment:
Why is it there? (in that PR?)

Reply:
I needs to be fixed inn 6.2, it was removed in https://issues.rudder.io/issues/18424 and I think it causes some issues 

especially we don't parse hash when reloading page not displaying the request anymore